.github/workflows/release.yaml

```
@@ -135,9 +136,25 @@ jobs:
           merge-multiple: true
       - uses: ncipollo/release-action@v1
         with:
-          artifacts: commitlint/commitlint-*.tar.gz
+          artifacts: commitlint/commitlint-*.tar.gz,schema.json
           generateReleaseNotes: true
 
+  schema:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-rs/toolchain@v1
```

actionlint (1.7.4)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)